IDE/GCC-ARM/Makefile.common

```diff
@@ -113,18 +114,28 @@ SRC_C += ../../wolfcrypt/src/wolfcrypt_last.c
 endif
 
 # wolfCrypt Additional
+SRC_C += ../../wolfcrypt/src/asm.c
+SRC_C += ../../wolfcrypt/src/async.c
 SRC_C += ../../wolfcrypt/src/asn.c
+SRC_C += ../../wolfcrypt/src/blake2s.c
 SRC_C += ../../wolfcrypt/src/chacha.c
-SRC_C += ../../wolfcrypt/src/cmac.c
+SRC_C += ../../wolfcrypt/src/chacha20_poly1305.c
 SRC_C += ../../wolfcrypt/src/coding.c
 SRC_C += ../../wolfcrypt/src/compress.c
 SRC_C += ../../wolfcrypt/src/cpuid.c
 SRC_C += ../../wolfcrypt/src/cryptocb.c
 SRC_C += ../../wolfcrypt/src/curve25519.c
+SRC_C += ../../wolfcrypt/src/curve448.c
+SRC_C += ../../wolfcrypt/src/ecc_fp.c
+SRC_C += ../../wolfcrypt/src/eccsi.c
 SRC_C += ../../wolfcrypt/src/ed25519.c
+SRC_C += ../../wolfcrypt/src/ed448.c
 SRC_C += ../../wolfcrypt/src/error.c
+SRC_C += ../../wolfcrypt/src/evp.c
```

Comment:
Same for `evp.c`. Technically no need to compile directly, since it is included via another .c. There are some macros protections to allow this, but you may want to make sure WOLFSSL_IGNORE_FILE_WARN is defined in the user_settings.h.

Reply:
added to user_settings